venom_core/api/audio_stream.py

```
@@ -848,7 +857,51 @@ def _multi_runtime_runtime_snapshot(self) -> dict[str, Any]:
+                version = _coerce_str(data.get("component_snapshot_version"), "")
+                timestamp_ms = _coerce_int(
+                    data.get("component_snapshot_timestamp_ms"),
+                    0,
+                )
+                payload = {
+                    "live_component_snapshot_version": version or None,
+                    "live_component_snapshot_timestamp_ms": (
+                        timestamp_ms if timestamp_ms > 0 else None
+                    ),
+                }
+        except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)